cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/customaction/DefaultCustomActionConfigurer.java

```diff
@@ -42,7 +46,9 @@
   private String description;
   private Map<String, String> properties;
 
-  private DefaultCustomActionConfigurer(CustomAction customAction) {
+  public DefaultCustomActionConfigurer(CustomAction customAction, Id.Namespace deployNamespace, Id.Artifact artifactId,
```

Comment:
This is used by `WorkflowNodeCreator` to create the custom action node. So added it as `public` similar to the `DefaultWorkflowActionConfigurer`.